client/blocks/inline-help/inline-help-support-article/index.jsx

```diff
@@ -0,0 +1,71 @@
+/** @format */
+/**
+ * External Dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import EmbedContainer from 'components/embed-container';
+import SupportArticleHeader from 'blocks/inline-help/inline-help-support-article/header';
+import Placeholders from 'blocks/inline-help/inline-help-support-article/placeholders';
+import { getSite } from 'state/reader/sites/selectors';
+import QueryReaderSite from 'components/data/query-reader-site';
+import QueryReaderPost from 'components/data/query-reader-post';
+import Emojify from 'components/emojify';
+import { getPostByKey } from 'state/reader/posts/selectors';
+import QueryPostLikes from 'components/data/query-post-likes';
+
+export class FullPostView extends React.Component {
+	static propTypes = {
+		post: PropTypes.object,
+	};
+
+	render() {
+		const { post, site, blogId, postId } = this.props;
+		const isLoading = ! post;
+		const postKey = { blogId, postId };
+
+		/*eslint-disable react/no-danger */
+		return (
+			<div className="inline-help-support-article">
+				{ site && <QueryPostLikes siteId={ post.site_ID } postId={ post.ID } /> }
+				{ post && post.site_ID && <QueryReaderSite siteId={ +post.site_ID } /> }
```

Comment:
The `propType` of `siteId` is a number currently and so we either do the coercing here or update. the propType to be a string (or string/number) and then change the type. It did make me wonder - should it not just be a string? We'll not be doing any arithmetic on it anyway 🤷‍♂️ 

 I thought to leave it as is, mostly because any change to `QueryReaderSite` would then require change elsewhere and since this is a second-class version of the read component, I'd rather do a minimal amount of change.